code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -41,9 +41,11 @@
 	description = "Slightly reduces stun times. If overdosed it will deal toxin and oxygen damage."
 	reagent_state = LIQUID
 	color = "#60A584" // rgb: 96, 165, 132
-	addiction_threshold = 30
+	addiction_threshold = 5
 	taste_description = "smoke"
 	trippy = FALSE
+	overdose_threshold=10
+	metabolization_rate = 0.115 * REAGENTS_METABOLISM
```

Comment:
I suggest using 0.125 instead of 0.115, so metabolism becomes 0.05 per tick instead of 0.046 (which hurts my math-autism).

Reply:
Probably will later today, I tried .125 but for some reason decided against it so I’ll see the reason when I adjust it.

Reply:
ok I did the thing